fastapi-app/schemas/recipe.py

```diff
@@ -0,0 +1,29 @@
+from uuid import UUID
+from datetime import datetime
+from pydantic import BaseModel
+from typing import Optional, List
+
+
+class RecipeBase(BaseModel):
+    class Config:
+        orm_mode = True
+
+
+class RecipeCreate(RecipeBase):
+    name: str
+    preparation: str
+    ingredients: dict
+
+
+class Recipe(RecipeBase):
+    id: UUID
+    name: str
+    preparation: str
+    ingredients: 'List[Ingredient]'
+    created_at: datetime
+    modified_at: Optional[datetime]
+
+
+from .ingredient import Ingredient  # noqa
```

Comment:
Why not on top of file and why # noqa?

Reply:
This is due to pydantic schema import issue, he might get an error if he puts it on top but I'm not 100% sure.